test/e2e/gatekeeper_test.go

```diff
@@ -65,6 +65,13 @@ var _ = Describe("Test gatekeeper", Ordered, func() {
 			By("Checking number of pods in gatekeeper-system ns")
 			utils.ListWithTimeoutByNamespace(clientManagedDynamic, common.GvrPod, metav1.ListOptions{}, "gatekeeper-system", 6, true, 240)
 		})
+
+		AfterAll(func() {
+			if CurrentSpecReport().Failed() {
+				utils.KubectlWithOutput("-n", "gatekeeper-system", "get", "pods", "--kubeconfig="+kubeconfigManaged)
```

Comment:
Side note: We should probably put that Gatekeeper namespace in a variable in another PR.